test/csv/csv_preserve_trailing_spaces.json

```diff
@@ -0,0 +1,15 @@
+{
+    "name": "Get existing CSV file with trailing spaces in `string,array`",
+    "request": {
+        "server_url": "{{ datastore "req_base_url" }}",
+        "endpoint": "bounce-json",
+        "method": "POST",
+        "body": {{ file_csv "../_res/assets/trailing_spaces_in_string_array.csv" ',' | marshal }}
+    },
+    "response": {
+        "statuscode": 200,
+        "body": {
+            "body": [{"id":0,"notes":["No surrounding","spaces"]},{"id":1,"notes":[" Leading","space"]},{"id":2,"notes":["Trailing ","space"]},{"id":3,"notes":[" Sorrounding ","spaces"]}]     
```

Comment:
You can use a formatted json here for better readability. the apitest tool tests this as json not as string